games/gopsHelpers.js

```diff
@@ -72,17 +78,78 @@ function roundWinner(instance) {
   }
 };
 
+//Awards points to the winner of the round. Takes roundWinner as an argument.
 function awardPoints(instance, id) {
 
   if (id) {
     instance[id].points += instance.prizePool[0];
     instance.prizePool.splice(0, 1);
+    reset(instance);
   } else {
     let message = "Round nulled, two or more players played same card."
     instance.prizePool.splice(0, 1);
+    reset(instance);
+    return message;
   }
 }
 
+//Once winner has been decided, resets hand of each player to zero.
+function reset(instance) {
+
+  for (let player in instance) {
+    if(instance[player].readyCard !== undefined){
+      instance[player].readyCard = 0
+    }
+  }
+};
+
+//At the begginning of every round, will check to see if every player has played. Status conditional -- needs to run
+//before checking a winner.
+function readyCheck(instance) {
+
+  let ready = true;
+  for (let player in instance) {
+    if(instance[player].readyCard === 0) {
+      ready = false;
+    }
+  }
+  return ready;
+}
+
+//Cheecks to see if the game is still playable based on the number of cards in the prize pool. At zero, returns false.
+function gameCheck(instance){
+
+  let status = true;
+  if (instance.prizePool.length === 0) {
+    status = false
```

Comment:
same as above, simply `return false` and then outside the if (no need for an else), put `return true`